tests/switch/bench-switch.c

```diff
@@ -32,7 +32,8 @@
 
 void test_benchmark_switch(int argc, char **argv);
 
-void test_benchmark_switch(int argc, char **argv)
+static void test_benchmark_switch_with_edges(int argc, char **argv, int west_max,
```

Comment:
not a real problem, but I'm not a big fan of this name, `switch_benchmark` would be enouth